csharp/src/Microsoft.ML.OnnxRuntime/NativeOnnxTensorMemory.cs

```diff
@@ -20,8 +20,6 @@ internal interface NativeMemoryHandler : IDisposable
 
     internal class NativeOnnxTensorMemory<T> : MemoryManager<T>, NativeMemoryHandler
     {
-        private bool _disposed;
-        private int _referenceCount;
```

Comment:
What's the reason for removing _disposed? Whilst it's not strictly necessary given you can test if the handle is null, it is useful when debugging to know if Dispose has been called.

Reply:
The property IsDisposed still works. The debugger will show the handle. Instead of having a bunch of confusing if() that do not do anything, I just have one that matters.

---
In reply to: [476749754](https://github.com/microsoft/onnxruntime/pull/4915#discussion_r476749754) [](ancestors = 476749754)